example/lib/database.dart

```diff
@@ -1,13 +1,17 @@
 import 'dart:async';
 
+import 'package:example/mail.dart';
+import 'package:example/mail_dao.dart';
 import 'package:example/task.dart';
 import 'package:example/task_dao.dart';
 import 'package:floor/floor.dart';
 import 'package:sqflite/sqflite.dart' as sqflite;
 
 part 'database.g.dart';
 
-@Database(version: 1, entities: [Task])
+@Database(version: 1, entities: [Task, Mail])
```

Comment:
💡 I would suggest keeping the example app as simple as possible. Thereby, we could move the `Mail` entity and `MailDao` into the integration test directory of floor. Additionally, it would be nice to have some actual integration tests that assert the functionality of FTS. You can find some examples here https://github.com/vitusortner/floor/tree/develop/floor/test/integration.

Reply:
i delete mail and maildao on new commit
next time i will write same actual integration tests that assert the functionality of FTS